package-lock.json

```diff
@@ -0,0 +1,6094 @@
+{
```

Comment:
We use `pnpm` instead of `npm` for package management, so the package-lock file shouldn't commit.